content/getting-started/linux.md

```diff
@@ -122,6 +122,9 @@ sudo apt-get update
 sudo apt-get install clang-8 llvm-8-dev lld-8 libclang-8-dev
 ```
 
+For Arch Linux, install [the llvm package](https://www.archlinux.org/packages/extra/x86_64/llvm/).
+It must be version 8.0.0 or above.
```

Comment:
> It must be version 8.0.0 or above.

But it must be LLVM 8 (8.1.0 would be OK, if that gets released).

Reply:
I wanted to illustrate anything => 8.0.0, is the strict requirement  8.0.0 <= x < 9.0.0?

Reply:
Oops, never replied to this.
Yes, the current requirement is on 8.x.x, so `8.0.0 <= x < 9.0.0`.

Reply:
Updated